pkg/hessian2/math/math.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 CloudWeGo Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package math
+
+import gxbig "github.com/dubbogo/gost/math/big"
+
+type BigType interface {
+	// String returns string representation of specific java big type(e.g. BigInteger, BigDecimal)
+	String() string
+}
+
+func NewBigDecimalFromString(str string) (BigType, error) {
+	return gxbig.NewDecFromString(str)
+}
```

Comment:
Is there any other options from standard lib to get big decimal from string ?

Reply:
This PR should be closed.

Actually, we have to use the type provided by dubbogo if we decide to maintain the ```BigType```.
For now, it has been solved by idl-ref functionality.